blog/02-02-2023-connect-and-interact-with-celo-using-web3onboard-library/connect-and-interact-with-celo-using-web3onboard-library.md

```diff
@@ -0,0 +1,1106 @@
+          result.receipt = rec;
+          result.view = false;
+          if(cancelLoading) cancelLoading();
+        });
+        break;
+
+      case 'stakeOnBehalf':
+        const txn_1 = await vault_ins.stakeOnBehalf(who, {value: value });
+        await txn_1?.wait(2).then((rec: ContractReceipt) => {
+          result.receipt = rec;
+          result.view = false;
+          if(cancelLoading) cancelLoading();
+        });
+        break;
+      
+      case 'unstake':
+        const txn2 = await vault_ins.unstake();
+        await txn2?.wait(2).then((rec: ContractReceipt) => {
+          result.receipt = rec;
+          result.view = false;
+          if(cancelLoading) cancelLoading();
+        });
+        break;
+
+      case 'withdraw':
+        const txn3 = await vault_ins.withdraw();
+        await txn3?.wait(2).then((rec: ContractReceipt) => {
+          result.receipt = rec;
+          result.view = false;
+          if(cancelLoading) cancelLoading();
+        });
+        break;
+
+      case 'getStakeProfile':
+        await vault_ins.getStakeProfile().then((res: Profile) => {
+          result.read = res;
+          result.view = true;
+          if(cancelLoading) cancelLoading();
+        });
+        break;
+
+      case 'balance':
+        await token_ins_noSigner.balanceOf(account).then((res: ethers.BigNumber) => {
+          result.read = res;
+          result.view = true;
+          if(cancelLoading) cancelLoading();
+        });
+      break;
+
+      default:
+        break;
+      }
+      
+    } catch (error: any) {
+      if(cancelLoading) cancelLoading();
+      if(error){
+        if(error.data?.code === -32000) alert("Please fund your wallet with test Celo from https://faucet.celo.org");
+      }
+    }
+  return result;
+};
+
+export default sendtransaction;
+```
+
+## Conclusion​
+
+Thumbs up for seeing it to the end. So far, we have learned:
+
+- Writing, compiling, deploying and testing smart contracts using foundry.
+- How to install and set up web3Onboard library.
+- How to build a simple frontend and connect the backend.
+- Manage your dApp using web3Onboard.
+
+## What next?
+​
+You can edit the code to your taste. At this point, you should be able to launch your own project as Celo developer. If you are confused anywhere in the tutorial, I left a link to the full source code at the _references_ section. Go over it as many times as you can. In no time, you will get a good grasp. Celo is EVM-compatible blockchain. You can leverage the various web3 tools to build your dream application. Get started with **[Celo documentation](https://docs.celo.org/tutorials)**
+
+# About the Author​
+
+**Isaac Jesse** , aka _Bobelr_ is a smart contract/Web3 developer. He has been in the field since 2018, worked as an ambassador with several projects like Algorand and so on as content producer. He has also contributed to Web3 projects as a developer.
+
+# References​
```

Comment:
Use ## instead of #

Reply:
Please read the guidelines for review. This is the correct format for the headings.